lib/cc/engine/analyzers/python/main.rb

```diff
@@ -12,11 +12,20 @@ module Python
         class Main < CC::Engine::Analyzers::Base
           LANGUAGE = "python"
           DEFAULT_PATHS = ["**/*.py"]
-          DEFAULT_MASS_THRESHOLD = 40
-          BASE_POINTS = 1000
+          DEFAULT_MASS_THRESHOLD = 32
+          BASE_POINTS = 1_500_000
+          POINTS_PER_OVERAGE = 50_000
+
+          def calculate_points(mass)
+            BASE_POINTS + (overage(mass) * POINTS_PER_OVERAGE)
+          end
 
           private
 
+          def overage(mass)
```

Comment:
> The calculate_points method hasn't been moved yet because 2 of the 4 languages use the old calculate_points formula (base * mass)

Is that intentional? Is the card you're working on fix "Duplication" or fix "Python"?